client/src/components/GenderCategory.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from 'styled-components';
+import Gender from './Gender.jsx';
+import Category from './Category.jsx';
+
+const GenderCategory = styled.div`
+  display: flex;
+  flex-flow: row;
+  font-family: Helvetica;
+  font-size: 1.5em;
+  text-transform: capitalize;
+  Color: #000000;
+  background-Color: pink;
+`;
+
+export default ({ gender, category }) => {
+  console.log(gender, category);
+  return (
+    <GenderCategory>
```

Comment:
Why have a separate component for Gender and Category? This is one string on the Nike.com page, i.e. "Men's Shoe"

I would suggest combining them together.

Reply:
Combined.